DP03_05_Using_User-Supplied_Catalogs.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>This cell had some errors from flake-8, attaching here (after INFO: firstnumber:secondnumber are as: firstnumber = line number, secondnumber = starting character number in line</p>

<pre class="ql-syntax" spellcheck="false">pycodestyle INFO: 2:78: W291 trailing whitespace
pycodestyle INFO: 3:66: W291 trailing whitespace
pycodestyle INFO: 4:80: E501 line too long (105 &gt; 79 characters)
pycodestyle INFO: 47: E203 whitespace before ':'
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>